### PR TITLE
Using single quote in locale files

### PR DESCRIPTION
**Pull request by allenwq**
For #136

### CODE REVIEW

File: config/locales/devise.en.yml [hunk 1]
```diff
@@ -3,61 +3,61 @@
...
       confirmation_period_expired: "needs to be confirmed within %{period}, please request a new one"
-      expired: "has expired, please request a new one"
-      not_found: "not found"
-      not_locked: "was not locked"
+      expired: 'has expired, please request a new one'
+      not_found: 'not found'
+      not_locked: 'was not locked'
       not_saved:
-        one: "1 error prohibited this %{resource} from being saved:"
+        one: '1 error prohibited this %{resource} from being saved:'
```
Comment (jsyeo) [inline config/locales/devise.en.yml]: This should be double quote.

Comment (allenwq) [inline config/locales/devise.en.yml]: Oh ! Sorry.

File: config/locales/devise.en.yml [hunk 2]
```diff
@@ -3,61 +3,61 @@
...
       invalid: "Invalid %{authentication_keys} or password."
-      locked: "Your account is locked."
-      last_attempt: "You have one more attempt before your account is locked."
+      locked: 'Your account is locked.'
+      last_attempt: 'You have one more attempt before your account is locked.'
       not_found_in_database: "Invalid %{authentication_keys} or password."
-      timeout: "Your session expired. Please sign in again to continue."
-      unauthenticated: "You need to sign in or sign up before continuing."
-      unconfirmed: "You have to confirm your email address before continuing."
+      timeout: 'Your session expired. Please sign in again to continue.'
```
Comment (jsyeo) [inline config/locales/devise.en.yml]: Should this be Your session *has* expired? I'm not sure. My English is bad.

**Comment by lowjoel** [comment]
@jsyeo sorry race condition

File: config/locales/devise.en.yml [hunk 3]
```diff
@@ -3,61 +3,61 @@
...
+      unauthenticated: 'You need to sign in or sign up before continuing.'
```
Comment (jsyeo) [inline config/locales/devise.en.yml]: I think sign in would suffice?

File: config/locales/devise.en.yml (hunk 2 above)
Comment (allenwq) [inline config/locales/devise.en.yml]: Looks like their english is not good, hah

Comment (jsyeo) [inline config/locales/devise.en.yml]: Oh? It's generated? LOL...

Comment (allenwq) [inline config/locales/devise.en.yml]: Yeah.generated...my english is even bad that them :(

**Comment by jsyeo** [comment]
@allenwq, the github issue will only be automatically closed if you put the issue number in the commit message, not in the PR description.

**Comment by allenwq** [comment]
@jsyeo, I put in the commits already

**Comment by jsyeo** [comment]
Oops. Just saw your commit message. :+1: my bad

**Comment by allenwq** [comment]
@jsyeo Thanks ! That was what you told me before, hope to get more github hints from you :+1:

**Comment by jsyeo** [comment]
My bad :sweat_smile:

**Pull request merged**